packages/expo-modules-core/ios/Arguments/AnyArgument.swift

```diff
@@ -33,14 +33,24 @@ extension Double: AnyArgument {}
 extension CGFloat: AnyArgument {}
 
 extension String: AnyArgument {}
-extension Dictionary: AnyArgument {}
+extension Data: AnyArgument {
```

Comment:
Nitpick: so far the extensions were ordered by "complexity", from primitive types to arrays, optionals, dictionaries – can you put it at the end of the file as it's a less common type?